break/viewABCD.md

```diff
@@ -0,0 +1,249 @@
+---
+jupyter:
+  jupytext:
+    formats: ipynb,md
+    text_representation:
+      extension: .md
+      format_name: markdown
+      format_version: '1.2'
+      jupytext_version: 1.7.1
+  kernelspec:
+    display_name: Python 3
+    language: python
+    name: python3
+---
+
+```python
+import pandas as pd
+import numpy as np
+import plotly.express as px
+import seaborn as sns
+import matplotlib.pyplot as plt
+import plotly.express as px
+
+import os
+from pathlib import Path
+from glob import glob
+from collections import namedtuple
+
+import requests
+```
+
+### Gather all the data elements from all the txt files
+
+```python
+files = sorted(glob("/home/jovyan/ABCD3/*.txt"))
```

Comment:
I think it might be a good idea to define a `data_path` at the top, so that users can change one variable when running elsewhere. Something like:

```python
data_path = Path("/home/jovyan/ABCD3")
files = sorted(data_path.glob("*.txt"))
```

Reply:
```suggestion
data_path = Path("/home/jovyan/ABCD3")
files = sorted(data_path.glob("*.txt"))
```